### ISSUE TITLE
logo proposal for Srver-tools

### ISSUE THREAD

**Issue by orlant** [OCA/server-tools#1309]
I am a great graphic designer, this project is a great tool and should have a great design that identifies it, if you allow me I can create this logo for your project would be a pleasure to collaborate

**Comment by pedrobaeza** [comment]
Hi, @orlant there a lot of modules in this repositories, so if you wan to design the module icon in this style: https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg, you're welcome.

**Comment by orlant** [comment]
that design is the main idea ... now I ask this repository created maintainer-tools need a logo ?? I can gladly show you the ideas, or is it no longer necessary?

**Comment by pedrobaeza** [comment]
No, that logo is already created, but each module has a function and can have an associated icon for that function. See other examples:

https://github.com/OCA/vertical-association/blob/11.0/membership_extension/static/description/icon.svg
https://github.com/OCA/server-tools/blob/11.0/database_cleanup/static/description/icon.png
https://github.com/OCA/partner-contact/blob/11.0/partner_external_map/static/description/icon.svg

I know that they are not the great source of creativity, but it's what we have/need.

**Comment by pedrobaeza** [comment]
I'm thinking that maybe you can design a logo for 2 things:

- OCB, which is a derivative of Odoo, but with extra patches and backports: https://github.com/OCA/OCB. Right now it uses the Odoo logo.
- Oxide, which is a distribution of Odoo + OCA addons (modules), which can be advertise with a new brand/logo.

What do you think?

**Comment by orlant** [comment]
It is very interesting, all the information you have given me, which is important to be able to start creating your logos, we will work with one first. tell me with what we started this first stage. where have I created this task request ?? And if you want some concept that you have in mind to take it into account in the creation? you think?

**Comment by orlant** [comment]
I have taken into account the moment when he told me that the design had to be under this concept since it is the main module of the project.

![oca](https://user-images.githubusercontent.com/40074778/42482963-b6ebce9a-83b8-11e8-944a-96b07d0d2504.png)

then just try to combine the cube of the main module with the OBC letters. And this was the result
![ocb12](https://user-images.githubusercontent.com/40074778/42483018-0a16052c-83b9-11e8-80d5-4f28fda4dede.png)

and from this main idea some variations in the color and form of logo and icon originated ... if you wish I can show you with other types of different sources.
![ocb111](https://user-images.githubusercontent.com/40074778/42483061-4c662380-83b9-11e8-95c2-66b36d18fa8a.png)
![portada](https://user-images.githubusercontent.com/40074778/42483068-56d93c08-83b9-11e8-93c9-f18a92473667.png)

**Comment by elicoidal** [comment]
@OCA/board

**Comment by mreficent** [comment]
@pedrobaeza I don't understand where would you put the OCB logo you asked for
